explainer.md

```diff
@@ -675,7 +698,7 @@ interface VRWebGLLayer : VRLayer {
 // Coordinate Systems
 //
 
-interface VRCoordinateSystem {
+interface VRCoordinateSystem : EventTarget {
```

Comment:
And it needs to add `VRCoordinateSystemEvent` and `VRCoordinateSystemEventInit` accordingly?